src/actions/users.js

```diff
@@ -593,15 +601,15 @@ export function getProfilesNotInChannel(teamId, channelId, page, perPage = Gener
     };
 }
 
-export function getMe() {
-    return async (dispatch, getState) => {
+export function getMe(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
         const getMeFunc = bindClientFunc(
             Client4.getMe,
             UserTypes.USER_REQUEST,
             [UserTypes.RECEIVED_ME, UserTypes.USER_SUCCESS],
             UserTypes.USER_FAILURE
         );
-        const me = await getMeFunc(dispatch, getState);
+        const me: $Subtype<ActionResult> = await getMeFunc(dispatch, getState);
```

Comment:
When `Client4.getMe` eventually gets annotated, how do we reflects its type accurately for clients of this action creator?

Reply:
Currently, can use it only if it is compatible with the `ActionResult: {data, error}` type. If the client's return type is compatible with `{data}`, this is not a problem.

But later ActionResult's data needs to be improved to infer the "actual" type of data.